TheTradeDeskCRMDestination.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Activating The Trade Desk CRM Destination in Adobe Experience Platform
+description: This document is designed to help you activate profiles for your Trade Desk campaigns for audience targeting and suppression based on CRM data.
+---
+
+# The Trade Desk - CRM
+
+
+## Overview {#overview}
+
+This document is designed to help you activate profiles for your Trade Desk campaigns for audience targeting and suppression based on hashed emails.  
+
+The Trade Desk does not directly handle the upload file of email addresses at any time nor does The Trade Desk store your raw (unhashed) emails.
+
+## Prerequisites {#prerequisites}
+
+Before you can activate segments to The Trade Desk, you must contact your TTD account manager to sign the CRM Onboarding contract. TTD will then give permission and share your advertiser ID to configure your destination.  
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## ID Matching Requirements (#id-matching-requirements)
```

Comment:
This section is listed under the Facebook - and specifically for hashed email addresses. I had assumed it was a requirement when choosing to use the platform when hashing. Please let me know if that is not the case and it can be removed. Thank you.